### DISCUSSION TITLE
Behavior in low input bitrate

### DISCUSSION THREAD

**Discussion by gregfr** [OvenMediaLabs/OvenMediaEngine#1091] [Help]
Greetings

I was wondering about the behavior of the video encoders when the input bitrate is lower than the target bitrate... In my recent test,  the player auto-switching feature went crazy (whatever the player), so I guess the actual output bitrate is smaller than the bitrate stated in the manifest?

What do you think?

Regards

**Comment by naanlizard** [comment]
Doesn't that ignore the VBR encoding option? That's what we recommend our streamers use. 

We don't transcode anything though

**Comment by Keukhan** [comment]
@gregfr 

I'll correct what I miscommunicated. 
Output bitrate is not related to input bitrate. However, depending on the complexity of the input video, the output is lower than the bitrate set by the user.

**Comment by getroot** [comment]
Most players I know switch streams based on the bitrate meta value in the manifest. However, if the input bitrate is very low, there is a possibility that a considerable error may occur in estimating the bandwidth by the player.

**Comment by gregfr** [comment]
Do you advise vbr or cbr? I thought cbr was better for live streaming? I'm having issues with the player's adaptive switching, so I was wondering about that possibility.
